src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java

```diff
@@ -153,10 +150,8 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
           env.getAccumuloClient().namespaceOperations().setProperty(namespace, property.getKey(),
               property.getDefaultValue());
         } catch (AccumuloException ex) {
-          if (ex.getCause() instanceof ThriftTableOperationException) {
-            ThriftTableOperationException ttoe = (ThriftTableOperationException) ex.getCause();
-            if (ttoe.type == TableOperationExceptionType.NAMESPACE_NOTFOUND)
-              return;
+          if (ex.getCause() instanceof TableNotFoundException) {
```

Comment:
Should this be NamespaceNotFoundException?

Reply:
I think so good catch.  Fixed in 6a87136